### ISSUE TITLE
Links have low contrast

### ISSUE THREAD

**Issue by DNin01** [scratchfoundation/scratch-www#7508]
### The problem

The current link color is `#4d97ff`, the primary motion color, usually against a white or light gray background. Unfortunately, this isn't enough to comply with the WCAG guidelines.

### Steps to reproduce
1. Visit any page (e.g., the homepage).
2. Find a link.

### How to fix it
Darken the link color, possibly along with other blue UI elements.

**Comment by DNin01** [comment]
Resolved by #7588.

**Comment by Secret-chest** [comment]
@DNin01 Sure? It does pass WCAG but https://scratch.mit.edu/discuss/topic/694496/?page=1

**Comment by mxmou** [comment]
> @DNin01 Sure? It does pass WCAG but https://scratch.mit.edu/discuss/topic/694496/?page=1

The new link color does have enough contrast with the background, but it would also need 3:1 contrast with surrounding text to actually pass WCAG. See https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.

> If content is conveyed through the use of colors that differ not only in their hue, but that also have a significant difference in lightness, then this counts as an additional visual distinction, as long as the difference in relative luminance between the colors leads to a contrast ratio of 3:1 or greater.

**Comment by mxmou** [comment]
There's no way to fix this by only changing the link color. A possible solution is to use a darker text color - `#292b38` is dark enough to pass the 3:1 contrast requirement.

**Comment by Secret-chest** [comment]
> There's no way to fix this by only changing the link color. A possible solution is to use a darker text color - #292b38 is dark enough to pass the 3:1 contrast requirement.


Still not very readable.

**Comment by DNin01** [comment]
> > There's no way to fix this by only changing the link color. A possible solution is to use a darker text color - #292b38 is dark enough to pass the 3:1 contrast requirement.
> 
> Still not very readable.

What's "not readable"? This would make the text color darker which increases contrast.

**Comment by DNin01** [comment]
Also, we should probably move this talk to a new issue.
